monday/query_joins.py

```diff
@@ -456,3 +456,14 @@ def get_complexity_query():
     }'''
 
     return query
+
+def delete_item_query(item_id):
```

Comment:
Please move this query to the section for Item Resource Queries at the top of the file.

Reply:
file updated